README.md

```diff
@@ -300,17 +300,15 @@ This is useful to check if the script is running and sending values to Venus OS.
 
 `/data/dbus-opendtu/install.sh` installs the service persistently (see above).
 
-This also activates the service, so you don't need to run `svcadm enable /service/dbus-opendtu` manually.
-
 ### How to restart
 
 `/data/dbus-opendtu/restart.sh` restarts the service - e.g. after a config.ini change.
 
 ### How to uninstall
 
-`/data/dbus-opendtu/uninstall.sh` stops the service and prevents it from being restarted (e.g. after a reboot).
+`/data/dbus-opendtu/uninstall.sh` stops the service and prevents it from being restarted after a reboot. You have to reboot to stop the service. 
```

Comment:
Why this? Maybe add s.th. like "You have to reboot to stop the service **to be sure**."

Reply:
pushed a commit to fix that problem that led to this "wrong" sentence. And corrected readme.